server/node/src/paypalServerSdk.ts

```diff
@@ -1,9 +1,3 @@
-import { config } from "dotenv";
-import { join } from "path";
-
-const envFilePath = join(__dirname, "../../../", ".env");
-config({ path: envFilePath });
```

Comment:
Are we sure we can remove this code? I'm getting the "Missing API credentials" error when trying to run `npm start` from the server root 🤔